### ISSUE TITLE
Diagrams: stacked bars for investment

### ISSUE THREAD

**Issue by mstich0** [open-plan-tool/gui#210]
A stacked bar diagram to show total investment cost (especially for comparing different scenarios) similar like:

![Image](https://user-images.githubusercontent.com/105202247/200311720-d14532b3-d09e-463a-918d-d642d837bac0.png)

It should only show investments above 0

**Comment by Bachibouzouk** [comment]
total investment is capex_fix + capex_var * invested in capacity

This should only make sense for scenario comparison

**Comment by Bachibouzouk** [comment]
Pie chart if only one scenario (maybe)

**Comment by Bachibouzouk** [comment]
Other option of bar plot is total installed capacities

**Comment by Bachibouzouk** [comment]
@mstich0 - Should the replacement costs of an asset reaching its end-of-life during the project time be added to the total investment?

**Comment by mstich0** [comment]
Yes I would say so, in this logic also the selling price of a component (with remaining lifetime) at the end of a project should be included

**Comment by Bachibouzouk** [comment]
Should the total investment be annualized as well?

**Comment by mstich0** [comment]
As both have almost the informative value I´d argue we keep them annualized as they will be closer to the specific capex done in the input.

**Comment by c-moeller** [comment]
two diagram type should be displayed:
- stacked bar with total costs per component (PV, wind, battery, ....)
- stacked bar with cost types for total costs (CAPEX, OPEX fix, OPEX var, fuel)

**Comment by c-moeller** [comment]
calculate the costs in the post-processing:
- CAPEX total = installed capacity * specific annuity
- OPEX fix total = installed capacity * specific OPEX fix
- OPEX var total = sum flow * specific OPEX var
- fuel costs total = sum flow * specific fuel costs

**Comment by mstich0** [comment]
this is implemented
